grails-app/services/au/org/emii/portal/BulkDownloadService.groovy

```diff
@@ -145,4 +145,14 @@ class BulkDownloadService {
         zipStream.write bytes, 0, bytes.length
         zipStream.closeEntry()
     }
+
+    public static void copy(final InputStream input, final OutputStream output, final int bufferSize) throws IOException
```

Comment:
I'll try to make some sense out of that.

 * `org.apache.commons.io.IOUtils.copy` - Does not close streams when it's done - good for us, this was the original behaviour.
 * `org.codehaus.groovy.grails.io.support.IOUtils.copy` - Included with this grails version. Does close the streams when it's done - not good for us.

Options:
 * Implement stream copying (a few lines) like apache.commons.io
 * Pull `org.apache.commons.io` as a dependency. I've found it to be rather painful, but I'm not a maven/grails expert.